core/src/main/scripts/importer/metaImport.py

```diff
@@ -70,9 +70,15 @@ def interface():
                                        action='store_true',
                                        help='Skip tests requiring information '
                                             'from the cBioPortal installation')
-    parser.add_argument('-P', '--portal_properties', type=str,
-                        help='portal.properties file path (default: assumed hg19)',
-                        required=False) 
+    parser.add_argument('-species', '--species', type=str, default='human',
+                        help='species information (default: assumed human)',
+                        required=False)
+    parser.add_argument('-genome', '--reference_genome', type=str, default='hg19',
```

Comment:
What if a user enters `-genome hg18` but does not provide an arg for `-build` - Would `build` be set to default `37` or resolve to `38`.

Reply:
validateData.py script ensures the name of genome matches genome build.